### PR TITLE
sutra types added

### PR DESCRIPTION
**Pull request by hrishikeshrt**
(as per our discussion)

### CODE REVIEW

File: sutraani/1/1/1.1.71.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "i": "11071",
     "an": "स्वम्$11068##रूपम्$11068",
     "ad": "",
-    "ss": "अन्त्येन इता सह आदिः स्वस्य ",
+    "ss": "अन्त्येन इता सह आदिः स्वस्य रूपम्",
```
Comment (neeleshb) [inline sutraani/1/1/1.1.71.txt]: No, the original was correct. You can actually add the word "रूपस्य" at the end. Vibhaktis can change when words get copied to the other sutras.

Comment (neeleshb) [inline sutraani/1/1/1.1.71.txt]: No the original was correct, you can actually change this to रूपस्य. Vibhaktis can change when words move to other sutras.

Comment (hrishikeshrt) [reply sutraani/1/1/1.1.71.txt]: corrected

**Comment by neeleshb** [comment]
Thanks. There are a few changes needed in the data.txt file, but I am
merging this first, we can make those edits subsequently. Thanks for the
change. Will update the frontend in 2-3 days and will inform.

**Comment by neeleshb** [comment]
FYI, "At" is replaced with AT and "Ad" is replaced with AD to stay
consistent on case.
https://github.com/ashtadhyayi-com/data/commit/b1b6f25805dc36d2f07f950e31c1c823bd9d0fc3

**Comment by hrishikeshrt** [comment]
Alright :-)

**Pull request merged**